DfciPkg/SettingsManager/SettingsManagerProvider.c

```diff
@@ -359,21 +359,21 @@ ProviderValueAsAscii (
 
   switch (Provider->Type) {
     case DFCI_SETTING_TYPE_ENABLE:
-      ValueSize = sizeof (v);
+      ValueSize = sizeof (b);
       if (Current) {
-        Status = Provider->GetSettingValue (Provider, &ValueSize, &v);
+        Status = Provider->GetSettingValue (Provider, &ValueSize, &b);
       } else {
-        Status = Provider->GetDefaultValue (Provider, &ValueSize, &v);
+        Status = Provider->GetDefaultValue (Provider, &ValueSize, &b);
       }
 
       if (EFI_ERROR (Status)) {
         DEBUG ((DEBUG_ERROR, "Failed - GetSettingValue for ID %a Status = %r\n", Provider->Id, Status));
         break;
       }
 
-      if (v == ENABLE_INCONSISTENT) {
+      if (b == ENABLE_INCONSISTENT) {
         AsciiString = DFCI_STR_INCONSISTENT;
-      } else if (v) {
+      } else if (b) {

```

Comment:
`ENABLE_TRUE` is defined beside `ENABLE_INCONSISTENT`:

```C
#define ENABLE_TRUE          1
```

Is it strictly intended that enabled means `1`? If so, this should be:

```suggestion
      } else if (b == ENABLE_TRUE) {
```

Otherwise, other non-zero values will cause this to be true.

Reply:
Good catch, change added.